.github/workflows/ci.yml

```diff
@@ -367,7 +267,7 @@ jobs:
 
   test-windows:
     name: test-windows (${{ matrix.name }})
-    needs: [lint, protos, verify-vendor, go-gen, create-linux-boot-files]
```

Comment:
do we want to completely revert it or remove `create-linux-boot-files` from here?

Reply:
nvm, I see it's more than that